common/gobuild.py.launcher.go

```diff
@@ -89,12 +91,29 @@ func main() {
 			}
 		}
 		// get payload if not empty
-		var payload map[string]interface{}
+		isJsonObjectParam := true
+		var payloadForJsonObject map[string]interface{}
+		var payloadForJsonArray []interface{}
 		if value, ok := input["value"].(map[string]interface{}); ok {
-			payload = value
+			payloadForJsonObject = value
+		} else {
+			if value, ok := input["value"].([]interface{}); ok {
+				payloadForJsonArray = value
+				isJsonObjectParam = false
+			}
 		}
 		// process the request
-		result := action(payload)
+		var result interface{}
+		funcMain := reflect.ValueOf(Main)
+		if isJsonObjectParam {
+			param := []reflect.Value{reflect.ValueOf(payloadForJsonObject)}
+			reflectResult := funcMain.Call(param)
```

Comment:
If invokes Main directly, compile would be failed.  
So i used `go reflect` to invoke Main